packages/ui/src/components/TimelineItem/TimelineItem.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import { TimelineItem } from './TimelineItem';
+
+describe('TimelineItem', () => {
+  Object.defineProperty(window, 'matchMedia', {
```

Comment:
❓Would it break the test if you don't define it?

Reply:
Yes, it's necessary always when you use useBreakpointValue() in component. useBreakpoint() uses this props inside, thats why whenever we use useBreakPoint() and useBreakpointValue() we have to mock this, i can move Object.defineProperty to different function but... where ?